src/resources/packages/wizard/components/tabs/organizer/tab.tsx

```diff
@@ -107,7 +107,7 @@ const OrganizerContent = ({moveToNextTab, skipToNextTab}) => {
 			return true;
 		}
 
-		const emailPattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+		const emailPattern = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$|^$/;
```

Comment:
This seems far too compliated for an email regex. Use [this](https://github.com/the-events-calendar/event-tickets-plus/blob/1569f83b44c99f8f40b92d814928ff8c0f8df671/src/resources/js/waitlist/form.js#L57) ?

Reply:
There's nothing in the description or the ticket about changing this?
`|^$/` says "an empty string is valid.

That's not the way to validate it if empty - add a condition before the regex.
```php
if (!email) {
	return true;
}
```

If we did want to _simplify_ it, we could use `^[\w\-\.]+@([\w-]+\.)+[\w-]{2,}$` or the one that Dimi linked to.